tidb-cloud/premium/connect-via-private-connection-premium.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Connect to a TiDB Cloud Premium instance via AWS PrivateLink
+summary: Learn how to connect to your TiDB Cloud Premium instance via private endpoint with AWS.
+---
+
+# Connect to a TiDB Cloud Premium instance via AWS PrivateLink
+
+This document describes how to connect to your TiDB Cloud Premium instance via [AWS PrivateLink](https://aws.amazon.com/privatelink).
+
+> **Tip:**
+>
+> - To learn how to connect to a {{{ .starter }}} cluster via AWS PrivateLink, see [Connect to {{{ .starter }}} via AWS PrivateLink](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
+> - To learn how to connect to a TiDB Cloud Premium instance via private endpoint with Alibaba Cloud, see [Connect to a TiDB Cloud Premium instance via Alibaba Cloud Private Link](/tidb-cloud/connect-via-private-connection-premium-on-alicloud.md).
```

Comment:
- Keep wording about Essential.
- Remove wording about Alibaba Cloud for now to avoid blocking this PR.
```suggestion
> To learn how to connect to a {{{ .starter }}} or {{{ .essential }}} cluster via AWS PrivateLink, see [Connect to {{{ .starter }}} or Essential via AWS PrivateLink](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
```